templates/docs/patterns/breadcrumbs.md

```diff
@@ -8,14 +8,24 @@ context:
 
 <hr>
 
-You can use the breadcrumbs pattern to indicate where the current page sits in
-the site's navigation. The separators between each item are added via CSS, so
-you don't have to include them manually.
+You can use the breadcrumbs pattern to indicate where the current page sits in the site's navigation.
+
+- A nav element with an `aria-label` "Breadcrumb" identifies the structure as a breadcrumb trail
+- The set of links is structured using an ordered list
+- The separators between each item are added via CSS, so you don't have to include them manually.
 
 <div class="embedded-example"><a href="/docs/examples/patterns/breadcrumbs/" class="js-example">
 View example of the breadcrumbs pattern
 </a></div>
 
+### Deprecated markup
+
+The breadcrumb markup has been updated for accessibility reasons. Support for the old markup shown in the example below may be removed in future versions:
+
+<div class="embedded-example"><a href="/docs/examples/patterns/breadcrumbs-deprecated/" class="js-example">
```

Comment:
I think it would be enough to just link to the example, not embed it.
One less CodePen to load, and less confusing which one to use.

Also, please add `<span class="p-label--deprecated">Deprecated</span>` below the "Deprecated marku" heading for better visibility.